src/preset_cli/api/clients/dbt.py

```diff
@@ -724,7 +724,7 @@ def get_accounts(self) -> List[AccountSchema]:
         """
         List all accounts.
         """
-        url = self.baseurl / "api/v2/accounts/"
+        url = self.baseurl / "api/v3/accounts/"
```

Comment:
Maybe we could define somewhere:

```python
V2_API = self.baseurl / "api/v2"
V3_API = self.baseurl / "api/v3"
```

Then here we could:

```python
url = V3_API / "accounts/"
```

Reply:
ohh, that's a good idea! I have to do a follow up on other dbt Cloud related issue and I can include this.